site/_docs/posts.md

```diff
@@ -182,6 +176,10 @@ Excerpt
 Out-of-excerpt
 {% endhighlight %}
 
+Completely disable excerpts by setting your `excerpt_separator` to `""`.
+
+Also, as with any output generated by Liquid tags, you can pass the `| strip_html` flag to remove any html tags in the output. This is particularly helpful if you wish to output a post excerpt as a `meta="description"` tag within the post `head`, or anywhere else having html tags along with the content is not desirable.
```

Comment:
[Liquid standard filter](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#standard-filters)